lib/menu.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'pie'
+require_relative 'mash'
+require_relative 'chips'
+
+class Menu
+  def initialize(dishes = {"pie" => Pie, "mash" => Mash, "chips" => Chips})
+    @dishes = dishes
+    @menu_length = dishes.length
+  end
+
+  def view
+    number = 0
+    @dishes.map { |_key, dish|
```

Comment:
I like the use of map to display the menu, very clear to follow what's happening.